pkg/controller/update_cluster.go

```diff
@@ -26,20 +26,22 @@ import (
 
 	myspec "github.com/m3db/m3db-operator/pkg/apis/m3dboperator/v1"
 	"github.com/m3db/m3db-operator/pkg/k8sops"
+	"github.com/m3db/m3db-operator/pkg/k8sops/labels"
 	"github.com/m3db/m3db-operator/pkg/m3admin"
+	"github.com/m3db/m3db-operator/pkg/util/eventer"
+	"go.uber.org/zap"
 
 	"github.com/m3db/m3/src/query/generated/proto/admin"
 	"github.com/m3db/m3cluster/placement"
 
-	"github.com/m3db/m3db-operator/pkg/util/eventer"
+	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/labels"
+	klabels "k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/util/runtime"
 )
 
 const (
-	_isolationGroupKey = "isolation-group"
-	_zoneEmbedded      = "embedded"
+	_zoneEmbedded = "embedded"
```

Comment:
just out of curiosity, what do the underscores in the front of the const mean?

Reply:
I think our convention so far has been to prefix non-exported consts with an `_` to provide some extra distinction between what's a var vs. a const. I don't think it's standard Go, but I've seen it used in a few other Uber OSS repos and it's been our convention so far so just following that.